sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedSourceType.cs

```diff
@@ -5,22 +5,78 @@
 
 namespace Azure.AI.MetricsAdvisor.Models
 {
+    /// <summary>
+    /// The type of data source that ingests a <see cref="DataFeed"/> with data.
+    /// </summary>
     [CodeGenModel("DataSourceType")]
     public readonly partial struct DataFeedSourceType
```

Comment:
Not for this PR, but I'm noticing that most (not sure all) enums have the suffix `Type`.
Is this intentional? or a convention we are doing in .NET ?

Reply:
Not exactly intentional, but the "types" of things are usually enums. These properties have the same name across languages.